docs/make.bat

```diff
@@ -28,14 +28,19 @@ if "%1" == "" goto help
 
 :process_targets
 if "%1" == "clean" (
-	@echo Removing auto-generated files under 'docs' and 'src'...
+	echo Removing auto-generated files...
 	rmdir /S /Q %BUILDDIR%
 	del /Q %SOURCEDIR%\api_index.rst
     rmdir /S /Q %SOURCEDIR%\api\
 	rmdir /S /Q %SOURCEDIR%\examples\
+    del /Q %SOURCEDIR%\snippets\admonitions.md
 ) else (
-	@echo Generating API index...
```

Comment:
@ not really needed in front of `echo`, because of the `@ECHO OFF` at the top of `make.bat`.